### PR TITLE
Bump log4j from 1.2.17 to 2.17.2

### PR DESCRIPTION
**Pull request by vincentatcompass**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,7 +81,7 @@
         <commons-net-version>3.6</commons-net-version>
         <httpclient-version>4.5.10</httpclient-version>
         <jackson-databind-version>2.10.1</jackson-databind-version>
-        <log4j-version>1.2.17</log4j-version>
+        <log4j-version>2.17.2</log4j-version>
```
Comment (samshuster) [inline pom.xml:84]: thank you for catching this!

Does this build? I think the artifactId and groupId also need to be updated?
https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core

I can also take a look more into this tomorrow.

**Review by samshuster — COMMENTED** [review]
thank you for this contribution!

File: pom.xml (hunk 1 above)
Comment (vincentatcompass) [reply pom.xml:84]: hi there, no problem. cool project. i've updated the artifactId and groupId. but, it looks like there are some changes that are needed. Feel free to take over this where I have left it off tomorrow :)

Comment (samshuster) [reply pom.xml:84]: ok, i put up a new mr with the change using the log4j-1.2-api bridge. I am going to close out this mr. Thanks though for alerting!

**Pull request closed without merging**